upup/pkg/fi/nodeup/tags/tags.go

```diff
@@ -20,6 +20,10 @@ const (
 	TagOSFamilyRHEL   = "_rhel_family"
 	TagOSFamilyDebian = "_debian_family"
 
+	TagOSDebianJessie = "_jessie"
```

Comment:
I think we can revisit or improve these tags/consts, unless you've borrowed them from elsewhere in the project. IMO, it would be great if the names of the OS's also helped us understand which family they were in. IE: it's not totally obvious that centos8s has some relation to RHEL. Maybe we adopt a standard something like Family-Distro-Version. It's a nit, but I think these continue to be important choices.

Reply:
The tags are already in the Kops code:
https://github.com/kubernetes/kops/blob/1.16.0-alpha.2/nodeup/pkg/distros/distribution.go#L27

Not sure if this is the moment to change the tag format. I think the plan is to refactor this once we move to tar.gz package and don't depend on packaging to add more OSes:
https://github.com/kubernetes/kops/pull/7986#discussion_r357925711

Reply:
The tags are already in the Kops code:
https://github.com/kubernetes/kops/blob/1.16.0-alpha.2/nodeup/pkg/distros/distribution.go#L27

Not sure if this is the moment to change the tag format. I think the plan is to refactor this once we use more tar.gz package and don't depend on packaging to add more OSes:
https://github.com/kubernetes/kops/pull/7986#discussion_r357925711